src/plugins/saved_objects_management/public/services/namespace_service.ts

```diff
@@ -29,7 +29,7 @@ export interface SavedObjectsManagementNamespaceServiceStart {
 
 export class SavedObjectsManagementNamespaceService {
   private readonly namespaces = new Map<string, SavedObjectsManagementNamespace<unknown>>();
-  private readonly alias;
+  private alias: string = '';
```

Comment:
This changes the behavior a bit. Is there any error cases or edge case handling that needs to be changed to handle `''` vs `undefined` as the default value?